crates/matrix-sdk-indexeddb/src/event_cache_store/types.rs

```diff
@@ -94,6 +94,15 @@ impl Event {
             Event::OutOfBand(e) => e.relation(),
         }
     }
+
+    /// Sets the content of the underlying [`GenericEvent`]
```

Comment:
Is this the right name? it seems there's no such `GenericEvent` in this file.

Reply:
It's just a bit below, line [115](https://github.com/matrix-org/matrix-rust-sdk/blob/07b5d025d435ba03d87e8172ac9cbeb359783d0f/crates/matrix-sdk-indexeddb/src/event_cache_store/types.rs#L115)!